config/locales/client.en.yml

```diff
@@ -39,10 +39,27 @@ en:
         rank: "Rank"
       create: "Create"
       cancel: "Cancel"
+      close: "Close"
       delete: "Delete"
       edit: "Edit"
       back: "Back"
       save: "Save"
+      apply: "Apply"
+      recalculate: "Recalculate Scores"
+      recalculating: "Recalculating Scores..."
+      completed: "Successfully done!"
```

Comment:
Let's make the copy clearer, "Done! The scores have been successfully recalculated."